.github/ISSUE_TEMPLATE/documentation_improvement.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Documentation improvement
+about: Request a documentation update
+title: ''
+labels: 'C: Documentation'
+assignees: ''
+
+---
+
+**Describe the flaw**
+A clear description of what the error or weakness in the documentation is.
+
+**Suggest an improvement**
+A clear description of what you want to change or add to the documentation.
+
+**Supporting materials**
+If applicable, add links or screenshots to help explain the flaw.
+
+**Desktop/Environment**
```

Comment:
How much sense does this section make for documentation?

Reply:
It's relevant for installation instructions and documentation workflow matters, but less so for other topics. What do you suggest?

Reply:
Maybe we can rephrase this to specify the envrionment this is relevant for, e.g. documentation regarding MacOS...

Reply:
Done.